docs/en/reference/testing.rst

```diff
@@ -62,6 +62,33 @@ multiple concurrent database connections, transactions, locking, performance-rel
 In such cases, it is still important that a pull request fixing the issues is accompanied by a free-form reproducer
 that demonstrates the issue being fixed.
 
+Running Integration Tests locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default ``phpunit.xml.dist`` configuration file is set up to run the
+integration tests against SQLite, but you need to enable the extension
+``pdo_sqlite`` in your PHP configuration.
+
+To run the integration tests against another platform, you can use one
+of the configuration files used in our continuous integration setup.
+Those are stored under ``ci/github/``.
+
+For instance, to run tests against MySQL using the PDO driver, you
+should spin up a MySQL server, enable the ``pdo_mysql`` extension, and
+then run the following command:
+
+.. code-block:: console
+
+    $ phpunit -c ci/github/pdo_mysql.xml
+
+We do not currently have specific instructions on how to run a Database
+server, but we do recommend Docker as a convenient way to do so.
+To know which version of a database you should use, you can check
+``.github/workflows/continuous-integration.yml`` to see which versions
+we officially support. For instance, the
+supported PostgreSQL versions are listed under
+``jobs.phpunit-postgres.strategy.matrix.postgres-version``.
```

Comment:
_Supported_ versions are listed here: https://www.doctrine-project.org/projects/doctrine-dbal/en/4.0/reference/platforms.html

The CI config tells you what versions we _test against_.

Reply:
Addressed.